docs/pipelines.md

```diff
@@ -75,19 +83,26 @@ A `Pipeline` definition supports the following fields:
 - Optional:
   - [`resources`](#specifying-resources) - **alpha only** Specifies
     [`PipelineResources`](resources.md) needed or created by the `Tasks` comprising the `Pipeline`.
+  - [`params`](#specifying-parameters) - Specifies the `Parameters` that the `Pipeline` requires.
+  - [`workspaces`](#specifying-workspaces) - Specifies a set of Workspaces that the `Pipeline` requires.
   - [`tasks`](#adding-tasks-to-the-pipeline):
```

Comment:
`tasks` is missing `taskSpec` or `taskRef`, shall we add them here?

Reply:
added them :)